_IBC/dymension-axelar.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "dymension",
```

Comment:
not following alphabetical order (axelar must be chain_1)

Reply:
ditto for celestia and cosmoshub

Reply:
sorry, missed that rule in the description!
fixed.
take a look, plz

Reply:
Oh, these two file names also need to be swapped

Reply:
deleted duplicated file - hub-dym,
renamed:
 axl-dym
 cel-dym